eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -5,6 +5,7 @@ parameters:
 steps:
```

Comment:
For any changes in eng/common we should follow the workflow outlined at https://github.com/Azure/azure-sdk-tools/tree/master/eng/common#updating.